ddtrace/tracer.py

```diff
@@ -759,6 +731,34 @@ def _start_span(
                 p.on_span_start(span)
```

Comment:
I think that the appsec processor tries to override the sampling decision to manual keep here, however moving the sampling code down causes us to override their decision.

Reply:
Yeah, the appsec tests for Flask and Django expect `dd.p.dm=-4` (manual keep) and this change switched it to `dd.p.dm=-0` (default).

Reply:
Ok great, I see that it fixed the failing tests. @majorgreys are we good to resolve this?